src/main/java/com/minecolonies/client/gui/AbstractWindowSkeleton.java

```diff
@@ -34,12 +34,23 @@ public AbstractWindowSkeleton(final String resource)
      * @param id     Button ID.
      * @param action Consumer with the action to be performed.
      */
-    public final void registerButton(String id, Consumer<Button> action)
+    public final void registerButton(final String id, final Consumer<Button> action)
     {
         buttons.put(id, action);
     }
 
     /**
+     * Register a button on the window.
+     *
+     * @param id     Button ID.
+     * @param action Consumer with the action to be performed.
+     */
+    public final void registerButton(String id, Runnable action)
+    {
+        registerButton(id, (button) -> action.run());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the parentheses around the "button" parameter [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1611)